Lib/pyto_ui.py

```diff
@@ -4208,7 +4208,7 @@ def __ui_image_from_pil_image__(image):
         return None
 
     buffered = BytesIO()
-    image.save(buffered, format=image.format)
+    image.save(buffered, format='PNG')
```

Comment:
Is there a way to first test if __image.format__ is valid before ignoring it?

Reply:
It would of course be possible but I don’t think that it brings any benefit in this case. This parameter on the save method is useful if you want to ensure that PIL saves the image in the same format as it was opened from. Here however the purpose of saving the file is to generate a byte stream to be used for conversion to an UIImage.

I have no strong feelings about it - if the common wisdom is for checking so be it.